extensions/amp-bind/0.1/amp-state.js

```diff
@@ -146,7 +147,7 @@ export class AmpState extends AMP.BaseElement {
       return;
     }
     const id = user().assert(this.element.id, '<amp-state> must have an id.');
-    const state = Object.create(null);
+    const state = /** @type {!JsonObject} */ (map());
```

Comment:
The type is correct but returns `{}`, not a proto-less object. I'll see if I can change that in `dict()` itself in a follow-up.